src/main/java/core/basesyntax/models/animals/Animal.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.models.animals;
+
+public abstract class Animal {
+    public abstract void feed();
+
+    public void doSomething() {
+        System.out.printf("%s is chilling now...", this.getClass().getSimpleName());
```

Comment:
anyway we want to get this method implemented with custom logic in our subclasses, so it's better to leave it abstract

Reply:
ok, made it abstract